Description: Disable doctests due to missing fonts
 The expected fonts aren't included in the crate, so the tests fail.
Author: James McCoy <jamessan@debian.org>
Forwarded: not-needed
Last-Update: 2025-04-05 by Peter Michael Green
---
Index: ab-glyph/Cargo.toml
===================================================================
--- ab-glyph.orig/Cargo.toml
+++ ab-glyph/Cargo.toml
@@ -29,2 +29,3 @@
 [lib]
+doctest = false
 name = "ab_glyph"
